schema.sql

```diff
@@ -1,5 +1,31 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name varchar(100),
+    date_of_birth date,
+    escape_attempts int,
+    neutered boolean,
+    weight_kg decimal
 );
+
+-- ALTER TABLE animals 
+-- ADD species varchar(100)
+
+ALTER TABLE animals 
+DROP species;
+
+CREATE TABLE owners (
+  id serial PRIMARY KEY,
+  full_name varchar(100),
+  age int
+);
+
+CREATE TABLE species (
+  id serial PRIMARY KEY,
+  full_name varchar(100),
+  age int
+);
+
+ALTER TABLE animals
+  ADD species_id int
```

Comment:
- [ ] The `species_id` references the species table kindly add the referential integrity constraint.
e,g
``ALTER TABLE animals ADD species_id int REFERENCES species(id);``
- [ ] Its also a requirement that you add the column `owner_id` which is a foreign key referencing the owners table, kindly do so.